entc/integration/compose/gremlin-server/Dockerfile

```diff
@@ -3,7 +3,9 @@
 # in the LICENSE file in the root directory of this source tree.
 
 # Fetch base gremlin server image
-FROM tinkerpop/gremlin-server
+# This version compatible with Amazon Neptune Engine Version 1.0.5.0.
+# https://docs.aws.amazon.com/neptune/latest/userguide/engine-releases-1.0.5.0.html
+FROM tinkerpop/gremlin-server:3.4.11
 
 # Copy overriden server configuration.
 COPY gremlin-server.yaml tinkergraph-empty.properties /opt/gremlin-server/conf/
```

Comment:
Hey @seiichi1101 and thanks for contributing this fix.

The CI is failing, can you please debug why? I think you need to use this configuration file in the CI as well.

Reply:
I think we should pin this version as Seiichi did. And rebuild the `entgo/gremlin-server` image for CI.

Reply:
OK. I will debug why ci is failling.

Reply:
> The CI is failing, can you please debug why? 

This was because the configuration file is not attached to `tinkerpop/gremlin-server:3.4.11` in ci.
So, I modified the gremlin-server in ci to use the local configuration file `gremlin-server.yaml` and `tinkergraph-empty.properties`.

In this PR,  I changed not to use `entgo/gremlin-server` anymore because it looks unmaintained.

Reply:
The owner of `entgo/gremlin-server` is the Ent team.

cc: @alexsn

Reply:
Hi, @yonidavidson 

> Why not build a entgo/gremlin-server:3.4.11 at the beginning and use it the rest of the test?

Could you elaborate on that?